### PR TITLE
Create envd-on-remote-serve.md

### PR DESCRIPTION
**Pull request by aseaday**

### CODE REVIEW

**Comment by kemingy** [comment]
Please add this to https://github.com/tensorchord/envd-docs/blob/main/docs/.vitepress/config/sidebar/blog.ts

File: docs/blog/envd-on-remote-serve.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+author: 'nullday'
+avatar: 'https://www.github.com/aseaday.png'
+github: 'https://github.com/aseaday'
+introduction: 'nullday is one of the envd maintainers. He has a dog called Shoufu and mainly focus on how envd can help users to improve work and study.'
+---
+
+# 5 minutes to run your ML/AI environment on you remote machine
+
+Envd provided a strong way to run your machine learning environment. For some people, we may be provided a powerful server had a bundle of cuda devices. We also want our actual running happens on the remote server and be able to easily access the environment from our local machine.
```
Comment (gaocegege) [inline docs/blog/envd-on-remote-serve.md:10]: Please use `envd` here. We always use lower case.

File: docs/blog/envd-on-remote-serve.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Here is the the guide. 
+
+## For Linux Server
```
Comment (gaocegege) [inline docs/blog/envd-on-remote-serve.md:14]:
```suggestion
## For linux server
```

File: docs/blog/envd-on-remote-serve.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Make a directory or clone your repository, touch a file called `build.envd` and add the build script to this file. 
+
+Run `envd up` in your directory to start you environment.
+
+<img width="1040" alt="Screen Shot 2022-11-10 at 13 36 52" src="https://user-images.githubusercontent.com/3927355/201009233-d98f6c7b-10dc-4dfc-9329-68578a4154a4.png">
+
+
+### Access it from your local computer
+
+First, you can copy the private key access the environment/container to your local computer. on remote machine, open the `~/.ssh/config`, find the section whose host matched the directory you build your environment. For example, if you build your environment in the directory named `demo`, the host will be `host.envd`.
```
Comment (xieydd) [inline docs/blog/envd-on-remote-serve.md:76]: host.envd -> demo.envd

File: docs/blog/envd-on-remote-serve.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+### Access it from your local computer
+
+First, you can copy the private key access the environment/container to your local computer. on remote machine, open the `~/.ssh/config`, find the section whose host matched the directory you build your environment. For example, if you build your environment in the directory named `demo`, the host will be `host.envd`. 
+```
+Host demo.envd
+  ForwardAgent yes
+  PubkeyAcceptedKeyTypes +ssh-rsa
+  HostKeyAlgorithms +ssh-rsa
+  HostName 127.0.0.1
+  Port 39265
```
Comment (xieydd) [inline docs/blog/envd-on-remote-serve.md:83]: We can add some shell to get this port.

**Review by xieydd — COMMENTED** [review]
some nits

**Comment by aseaday** [comment]
> some nits

Thx and fixed

File: docs/blog/envd-on-remote-serve.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  UserKnownHostsFile /dev/null
+  IdentityFile "/root/.config/envd/id_rsa_envd" 
+  StrictHostKeyChecking no
+  ForwardAgent yes
```
Comment (xieydd) [inline docs/blog/envd-on-remote-serve.md:87]: Is this duplicated?
`ForwardAgent yes`, maybe is a bug of envd.
@gaocegege

Comment (gaocegege) [reply docs/blog/envd-on-remote-serve.md:87]: agent forwarding should work with docker daemon runner.

**Comment by gaocegege** [comment]
https://app.netlify.com/sites/envd/deploys/636cf7ca017cdd00085f5a80#L98

There is a build error.

File: docs/blog/envd-on-remote-serve.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+Then you can install envd by
+
+```bash
+pip install --pre --upgrade envd
+```
+There is a also a mirror for some China users:
+```bash
+pip install -i https://pypi.tuna.tsinghua.edu.cn/simple --pre --upgrade envd
+```
+Bootstarp the envd:
```
Comment (AlexXi19) [inline docs/blog/envd-on-remote-serve.md:42]: typo here

**Comment by aseaday** [comment]
> https://app.netlify.com/sites/envd/deploys/636cf7ca017cdd00085f5a80#L98
> 
> There is a build error.

fixed

File: docs/.vitepress/config/sidebar/blog.ts [hunk 7]
```diff
@@ -9,6 +9,7 @@ export const blogSidebar: DefaultTheme.Sidebar = {
         // eg: { text: 'Blog', link: '/blog/post1' },
         { text: 'Machine Learning Environment Should Be Easy', link: '/blog/ml-env' },
         { text: 'envd Is Participating Hacktoberfest!', link: '/blog/hacktoberfest' },
+        { text: '5 minutes to run your ML/AI environment on you remote machine', link: '/blog/envd-on-remote-serve' },
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar/blog.ts:12]:
```suggestion
        { text: '5 minutes to run your ML/AI environment on your remote machine', link: '/blog/envd-on-remote-serve' },
```

File: docs/blog/envd-on-remote-serve.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
+---
+author: 'nullday'
+avatar: 'https://www.github.com/aseaday.png'
+github: 'https://github.com/aseaday'
+introduction: 'nullday is one of the envd maintainers. He has a dog called Shoufu and mainly focus on how envd can help users to improve work and study.'
+---
+
+# 5 minutes to run your ML/AI environment on you remote machine
```
Comment (kemingy) [inline docs/blog/envd-on-remote-serve.md:8]:
```suggestion
# 5 minutes to run your ML/AI environment on your remote machine
```

**Review by kemingy — COMMENTED** [review]
typo

**Comment by gaocegege** [comment]
LGTM

**Pull request merged**